src/webgpu/shader/validation/parse/switch.spec.ts

```diff
@@ -0,0 +1,75 @@
+export const description = `Validation parser tests for 'switch' statements`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { keysOf } from '../../../../common/util/data_tables.js';
+import { ShaderValidationTest } from '../shader_validation_test.js';
+
+export const g = makeTestGroup(ShaderValidationTest);
+
+const kTests = {
+  L_default: { wgsl: `switch L { default {} }`, pass: true },
```

Comment:
Please add test cases that exercise parsing before the first paren
* `switch { default{} }`  // noexpr
* `switch (L { default {}}`  
* `switch L) { default {}}`
* `switch )L) { deafult {}}`
* `switch (L( { default {}}`
* `switch;`

Reply:
Done